style.css

```diff
@@ -0,0 +1,93 @@
+body {
+  display: flex;
+  background: url(background.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+}
+
+.wrapper {
+  max-width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.header {
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+  align-content: space-around;
+  margin-top: 2%;
+}
+
+.mylogo {
+  color: #6070ff;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-size: 18px;
+  font-weight: bold;
+  line-height: 20px;
+  flex: 1;
+  padding-left: 2%;
+}
+
+.icon {
+  align-self: center;
+  padding-right: 4%;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  margin-top: 20%;
+}
+
+.welcome {
+  color: #172b4d;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  display: flex;
+  margin: 3%;
+}
+
+p {
+  color: #344563;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 16px;
+  margin: 3%;
+  display: flex;
+  margin-top: 5%;
+}
+
+.connect {
+  color: #7f8cff;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  display: flex;
+  width: 100%;
+  margin: 3%;
+}
+
+ul {
+  list-style-type: none;
+  width: 55%;
+  display: flex;
+  margin: 3%;
+  padding: 0;
+  align-content: start;
+  align-items: center;
+}
+
+li {
+  flex: 1;
+}
+
+.social-media {
+  display: flex;
```

Comment:
### OPTIONAL
- [ ] For DRY css you should combine duplicate properties. Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.

- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and css best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)